### PR TITLE
Update the metadata and keywords for SEO

### PR DESCRIPTION
**Pull request by usha-mandya**
Updated the robots.txt and keywords in Docker Desktop docs for SEO.

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -33,4 +33,6 @@ Disallow: /datacenter/ucp/1.1
 
 Disallow: /apidocs/v*
 
+Disallow: /machine/
```
Comment (thaJeztah) [inline robots.txt]: Wasn't there an option to set this in the metadata of the files that should not be indexed? (that way we wouldn't have to update the robots.txt)

File: docker-for-mac/install.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 description: How to install Docker Desktop on Mac
-keywords: mac, beta, alpha, install, download
+keywords: mac, install, download, run, docker, local, machine
```
Comment (nebuk89) [inline docker-for-mac/install.md]: do we have a different page that will win download instead?

File: robots.txt (hunk 1 above)
Comment (usha-mandya) [reply robots.txt]: Not aware of the option to set the metadata of the file. Could you provide more details @thaJeztah?

File: docker-for-mac/install.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-mac/install.md]: Do you mean a different page where we provide Desktop downloads?

File: robots.txt (hunk 1 above)
Comment (thaJeztah) [reply robots.txt]: looks if we set `hide_from_sitemap` in the metadata; https://github.com/docker/docker.github.io/blob/24718c5e47eb85b7b219d34f13a4460df3616984/_layouts/docs.html#L105

File: docker-for-mac/install.md (hunk 2 above)
Comment (nebuk89) [reply docker-for-mac/install.md]: As discussed other pages that are better suited have the install tag

File: robots.txt (hunk 1 above)
Comment (usha-mandya) [reply robots.txt]: I guess it has to be set on a per-page level and not at the directory level? Any idea if affects both  docs.docker.com and Google search results? Thanks.

Comment (usha-mandya) [reply robots.txt]: Removed the update from robots.txt after discussing with @thaJeztah.

**Comment by usha-mandya** [comment]
Sanity checked Docker Machine pages in the Netlify preview to ensure the pages render as expected. They look good.
https://deploy-preview-10030--docsdocker.netlify.com/machine/overview/

**Comment by thaJeztah** [comment]
Thanks!

**Pull request merged**